src/app/questionGenerator.js

```diff
@@ -0,0 +1,44 @@
+class QuestionGenerator {
+
+    //argument argAmountOfId will be relaced by connection to CurrentMode and it's IDs'
+    static generateQuestionsForCurrentMode(amountOfId) {
+        const amountOfAnswersPerQuestion = 4;
+        let generatedIds = [];
+
+        if (amountOfId < amountOfAnswersPerQuestion) {
+            throw `amountOfId must be greater or equal ${amountOfAnswersPerQuestion}. Passed value: ${amountOfId}`
+        }
+
+        for (let index = 0; index < amountOfAnswersPerQuestion; index++) {
```

Comment:
Could you provide a sample of test that you ran to check the way that this function works?

Reply:
I used new project with simple files: index.html and index.js
Whole content of HTML:
<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <script type="module" defer src="./index.js"></script>
    <title>Document</title>
</head>

</html>

Content in index.js:
-Whole content of that push
-Function calls:
    const arrayWithId = QuestionGenerator.generateQuestionsForCurrentMode(17);
     const rightAnswer = QuestionGenerator.chooseCorrectAnswer(arrayWithId);
-console.log() for that variables: arrayWithId and rightAnswer
>